XamlControlsGallery/ControlPages/ListViewPage.xaml.cs

```diff
@@ -11,29 +11,58 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Drawing;
 using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
```

Comment:
Do we need System.Diagnostics and System.Reflection.MetaData ? 😅

Reply:
Thanks for pointing those out. Deleted.